### ISSUE TITLE
[RTM] If input provider call has parameter mark keep unused

### ISSUE THREAD

**Issue by baumannsven** [contao-community-alliance/dc-general#510]
https://github.com/contao-community-alliance/dc-general/blob/b5e7db300cb7ec00d907b9569a53c227d2f7ee12/src/Contao/InputProvider.php#L71

https://github.com/contao/contao/blob/9f8ec152650c02ac9b914280af98249186111439/core-bundle/src/Resources/contao/library/Contao/Input.php#L77

**Comment by baumannsven** [comment]
```php
public function hasParameter($key)
{
        return (null !== Input::get($key, false, true));
}
```

@discordier What do you think?

**Comment by discordier** [comment]
Yop, `hasParameter()` should keep the parameter unused.

**Comment by baumannsven** [comment]
Fixed in e69f4dd
